share/get.go

```diff
@@ -28,9 +28,14 @@ func GetShare(
 
 // GetLeaf fetches and returns the raw leaf.
 // It walks down the IPLD NMT tree until it finds the requested one.
-func GetLeaf(ctx context.Context, bGetter blockservice.BlockGetter, root cid.Cid, leaf, total int) (ipld.Node, error) {
+func GetLeaf(
```

Comment:
Let's move all the funcs return `format.Node` to the `ipld`. All the higher-order functions in `share` will just convert nodes/leaf into the actual shares defined in `share`.

Reply:
Good idea. I've splitted `GetShares` for that purpose and moved all logic related to `format.Node ` to `ipld` pkg